EosPayload/drivers/new_GPS_driver.py

```diff
@@ -1,38 +1,26 @@
+import serial
 import logging
-import queue
+import datetime
 import traceback
 
-import adafruit_gps
-import serial
-try:
-    import Adafruit_BBIO.UART as UART
-except ModuleNotFoundError:
-    pass
+from ublox_gps import UbloxGps
 
-from EosLib.device import Device
-from EosLib.format import Type
+from EosPayload.lib.base_drivers.position_aware_driver_base import PositionAwareDriverBase
 from EosLib.format.formats.position import Position, FlightState
-from EosLib.packet import Packet
-from EosLib.packet.data_header import DataHeader
+from EosLib.packet.packet import Packet, DataHeader
+from EosLib.device import Device
 from EosLib.packet.definitions import Priority
-import datetime
+from EosLib.format.definitions import Type
 
-from EosPayload.lib.base_drivers.position_aware_driver_base import PositionAwareDriverBase
 from EosPayload.lib.mqtt import Topic
 
 
-class GPSDriver(PositionAwareDriverBase):
+class NewGPSDriver(PositionAwareDriverBase):
```

Comment:
there was absolutely no point of renaming everything if you were deleting the old one ayway